localized/flags/png/.gitignore

```diff
@@ -0,0 +1,6 @@
+*.png
+!cn.png
+!de.png
+!fr.png
```

Comment:
Just a note to clear things up: This gitignore file (which has I removed in a later commit) seems to indicate that there are other flag files in that directory in your checkout, so local files. That's error-prone, because it means if a "rake clean build publish" on your machine is different than on other machines, as those files get copied into the _site directory to and uploaded (and potentially linked in sitemap.xml etc). If any page uses those local files, manual tests on your machine will look fine - convincing you it's ok to commit&push - but on other machines the site will break and if those publish it (or travis does), it will lead to a broken website in that case.

Due to this reason, we ignore nothing in the src directory. We only ignore only the target directory (which is the _site directory in this case) because that's generated, the IDE files because they can be generated from the build files and the local directory (which is local directory that isn't removed during "mvn clean" or "rake clean") which is the 1 exception to the rule above...